content/post/cheers-to-small-beginnings.md

```diff
@@ -0,0 +1,55 @@
+---
+author: ricardobaeta
+date: 2018-07-09
+title: "Cheers to small beginnings!"
+image: /post/cheers-to-small-beginnings/{d}ali-banner.png
+description: "A closer look at source{d}'s rebranding."
+categories: ["design", "branding"]
+draft: true
+---
+
+## A closer look at source{d}’s rebranding
+
+### Molto vivace
+
+We’re now moving fast on design as well. We have a design team of one, but we strongly believe that the design team it’s composed by every single member in our company. We’re very excited for the tools we’ve been building, and now is the time for design to jump onboard, with a humble goal, it’s not about how fast you ship, but how fast you learn, while shipping.
```

Comment:
Please break all paragraphs into smaller lines so reviews are easier to make.
I have 3 different comments on this single line, but if I make them it's gonna be a mess to followup.

Reply:
Thanks, already did it. Please be so kind to comment.